nmos-test.py

```diff
@@ -508,8 +514,19 @@ def validate_args(args):
 
 def start_web_servers():
     port = 5001
+    if ENABLE_HTTPS:
+        ctx = ssl.create_default_context()
+        ctx.options &= ssl.OP_NO_TLSv1 | ssl.OP_NO_TLSv1_1
+        for cert, key in zip(CERTS_MOCKS, KEYS_MOCKS):
+            ctx.load_cert_chain(cert, key)
+        # TODO: Provide a mechanism to establish trust for client certificates to avoid check_hostname=False

```

Comment:
Is there a specific case we need these exceptions for? I'm not aware that we're using client certificates anywhere at present.

Reply:
I don't think so - not for plain BCP-003-01. But we do need `check_hostname = False` in order to turn it off because it `create_default_context` has it on by default, so maybe it's worth a comment to explain?

Reply:
Yeah, if you think it's worth having in case of testing something beyond BCP-003-01 then it's probably worth something to mention it's for that purpose rather than something in the spec.

Reply:
What I mean is that TLS handshake fails if we _don't_ set it to `False`.
I'll rebase again having fixed these two things.

Reply:
How odd. I wouldn't have thought the server side would have a hostname to check against, unless it's validating its own one.

Reply:
I haven't really investigated how Client Authentication Certificates work, I just knew that we didn't want the server to require them!